index.html

```diff
@@ -101,11 +101,17 @@ <h2 id="community-collaboration-guide">Community Collaboration Guide</h2>
 
   <p>A Red Hat guide to assist enterprise projects in collaborating with upstream communities on documentation.</p>
 
+  <h2 id="supplementary-style-guide-for-product-documentation">Red Hat supplementary style guide for product documentation</h2>
+
+  <p><a href="https://redhat-documentation.github.io/supplementary-style-guide/#introduction">Red Hat supplementary style guide for product documentation</a></p>
```

Comment:
I'd suggest removing the #introduction anchor, so that users land at the very top of the page (with the title shown)

Reply:
Done.